server.js

```diff
@@ -16,10 +16,39 @@ connection.connect((err) => {
   }
 });
 
-app.get("/", (request, response) => {
-  response.send("Welcome to Express");
+app.get("/", (req, res) => {
+  res.send("Welcome to Express");
 });
 
+app.post("/create", (req,res) => {
```

Comment:
according to REST conventions, the path should represent the resource, not the action :)

so `/capsules` instead of `/create` (`/create` will be confusing when you will have to create playlist for example)